homeassistant/components/nsw_fuel_station/sensor.py

```diff
@@ -119,7 +119,7 @@ def extra_state_attributes(self) -> dict:
     @property
     def native_unit_of_measurement(self) -> str:
         """Return the units of measurement."""
-        return f"{CURRENCY_CENT}/{VOLUME_LITERS}"
+        return f"¢/{VOLUME_LITERS}"
```

Comment:
Would it be more appropriate for this to return $/L instead of c/L?  Note that the $ here is AUD not USD. It would pave a way forward for currency conversion later down the track (not that it would be super relevant for this as it's local to AU/NSW anyway)